libs/wlgen/wlgen/rta.py

```diff
@@ -266,19 +266,29 @@ def _confCustom(self):
 
         self.json = '{0:s}_{1:02d}.json'.format(self.name, self.exc_id)
         ofile = open(self.json, 'w')
-        ifile = open(self.params['custom'], 'r')
         replacements = {
             '__DURATION__' : str(self.duration),
             '__PVALUE__'   : str(calibration),
             '__LOGDIR__'   : str(self.run_dir),
             '__WORKDIR__'  : '"'+self.target.working_directory+'"',
         }
 
+	# check for inline config
+	if 'custom_config' in self.params['custom']:
+		# inline config present, turn it into a file repr so we can parse the same
+		tmp_json = json.dumps(self.params['custom']['custom_config'], indent=4, separators=(',', ': '), sort_keys=True)
+		ifile = tmp_json.split('\n')
+	else:
+		# open the supplied filename
+	        ifile = open(self.params['custom']['custom_filename'], 'r')
+
         for line in ifile:
             for src, target in replacements.iteritems():
                 line = line.replace(src, target)
+	    print "line: {}".format(line)
```

Comment:
Was this used just for debugging while modifying the code or should we actually print those lines?

In case, it is meant to be there, I have a couple of comments:
 - there's an indentation problem here.
 - could you use `logging.info()` (or `logging.debug()`)  instead of `print`?